async_lru/__init__.py

```diff
@@ -108,7 +109,17 @@ def __init__(
         self.__closed = False
         self.__hits = 0
         self.__misses = 0
-        self.__tasks: Set["asyncio.Task[_R]"] = set()
+
+    @property
+    def __tasks(self) -> List["asyncio.Task[_R]"]:
+        # NOTE: I don't think we need to form a set first here but not too sure we want it for guarantees
+        return list(
```

Comment:
I think a list is probably overkill, we could just return an iterator for all the uses I saw.